CODING_GUIDELINES.md

```diff
@@ -1,6 +1,7 @@
 # Coding Guidelines
 
 All new code **must** follow the following coding guidelines. \
```

Comment:
```suggestion
All new code **must** follow the following coding guidelines.
```

Reply:
Fair enough. This is my first contribution to an open source project so that's my bad. I should have double checked that I wasn't fat fingering it.

Regarding the project itself I look forward to getting familiar and contributing to actual code.